package.json

```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "preinstall": "npx only-allow pnpm"
```

Comment:
whats does it do?

Reply:
prevents from installing dependencies by something else than pnpm.